content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,243 @@
+falco-falcosidekick-ui-7c5fc8dd54-q4qh9   1/1     Running   0          2m16s
+falco-vkl4f                               1/1     Running   0          2m16s
+```
+
+## Test
+
+To test this, we have to connect a shell within the container.
+
+Let's list the pods that we already have.
+
+```shell
+$ kubectl get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+alpine-deployment-77789455d6-m7flp   1/1     Running   15         3h6m
+alpine-deployment-77789455d6-v7fkw   1/1     Running   15         3h6m
+podinfo-6df788c7b8-gs5qb             1/1     Running   0          3h28m
+podinfo-6df788c7b8-sfxvd             1/1     Running   0          3h28m
+```
+
+Now, run the following command to connect a shell.
+
+```shell
+$ kubectl exec -ti alpine-deployment-77789455d6-m7flp -- sh -c "uptime"
+19:35:58 up  3:04,  load average: 3.12, 1.91, 1.22
+```
+
+Once you run the command above, Falco will detect that malicious behavior and send it to the Falcosidekick via HTTP.
+
+You should see the similar output below in the `Falco` logs:
+
+```shell
+{"output":"19:35:58.532086161: Notice A shell was spawned in a container with an attached terminal (user=root user_loginuid=-1 k8s.ns=default k8s.pod=alpine-deployment-77789455d6-m7flp container=788861c3cf83 shell=sh parent=runc cmdline=sh -c uptime terminal=34816 container_id=788861c3cf83 image=alpine) k8s.ns=default k8s.pod=alpine-deployment-77789455d6-m7flp container=788861c3cf83","priority":"Notice","rule":"Terminal shell in container","time":"2021-06-13T19:35:58.532086161Z", "output_fields": {"container.id":"788861c3cf83","container.image.repository":"alpine","evt.time":1623612958532086161,"k8s.ns.name":"default","k8s.pod.name":"alpine-deployment-77789455d6-m7flp","proc.cmdline":"sh -c uptime","proc.name":"sh","proc.pname":"runc","proc.tty":34816,"user.loginuid":-1,"user.name":"root"}}
+```
+
+and the similar output below in the `Falcosidekick` logs:
+
+```shell
+falco-falcosidekick-5854669c76-ddvrv falcosidekick 2021/06/13 19:51:45 [INFO] : CloudEvents - Send OK
+```
+
+and the similar output below in the `falco-event-listener` logs:
+
+```shell
+falco-event-listener falco-event-listener 2021/06/13 19:35:59 resp.Status=200 OK
+falco-event-listener falco-event-listener 2021/06/13 19:35:59 resp.StatusCode=200
+falco-event-listener falco-event-listener 2021/06/13 19:35:59 [Terminal shell in container] scaled down to zero alpine-deployment-77789455d6-m7flp from default because 19:35:58.532086161: Notice A shell was spawned in a container with an attached terminal (user=root user_loginuid=-1 k8s.ns=default k8s.pod=alpine-deployment-77789455d6-m7flp container=788861c3cf83 shell=sh parent=runc cmdline=sh -c uptime terminal=34816 container_id=788861c3cf83 image=alpine) k8s.ns=default k8s.pod=alpine-deployment-77789455d6-m7flp container=788861c3cf83
+```
+
+You should also notice that there is a new commit in the `desired-state-repository` like the following picture:
```

Comment:
You should also notice that a new commit is available in the `desired-state-repository` as given below:

Reply:
thank you so much for fixing the issues, done.